ui-tests/src/test/java/org/openmrs/reference/ManageProviderSchedulesTest.java

```diff
@@ -39,7 +39,6 @@
 	@Before
 	public void setUp() throws Exception {
 		homePage = new HomePage(page);
-		assertPage(homePage);
```

Comment:
Since the homePage is already instantiated by the object _homePage = new HomePage(page)_ then there is no need to assertPage(homePage)